sys_siglist was removed from glibc. strsignal() is already used.

Upstream-Status: Inappropriate [OpenXT]

--- a/main.cpp
+++ b/main.cpp
@@ -51,7 +51,7 @@
 // ============================================================================
 
 void exitHandler(int signal) {
-    qDebug().nospace() << "Received " << sys_siglist[signal] << " (" << signal << ") : " << strsignal(signal);
+    qDebug().nospace() << "Received signal " << signal << " : " << strsignal(signal);
     qDebug() << "Exiting";
     QCoreApplication::quit();
 }
